tensorrt_llm/bench/benchmark/throughput.py

```diff
@@ -98,6 +109,12 @@
     required=False,
     help="Pass in a dataset file for parsing instead of stdin.",
 )
+@optgroup.option(
+    "--no_skip_tokenizer_init",
```

Comment:
Although it is clear what this technically does, can we document why it's useful to control this?